pkgs/by-name/li/libvisual/package.nix

```diff
@@ -2,19 +2,15 @@
   lib,
   stdenv,
   fetchFromGitHub,
-  SDL,
+  SDL_compat,
   autoreconfHook,
   autoconf-archive,
   glib,
-  pkg-config,
+  pkgconf,
   fullVariant ? false,
   withLvTool ? fullVariant,
   withExamples ? fullVariant,
 }:
-
-# Remove when 0.5.x is published.
-assert stdenv.hostPlatform != stdenv.buildPlatform -> !withLvTool;
```

Comment:
We just added the `# Remove when 0.5.x is published.` in the last commit. We might as well don't add it to have a cleaner history.